apisix/core/config_local.lua

```diff
@@ -55,6 +55,7 @@ end
 -- local local_conf = core.config.local_conf()
 -- local fallback_sni = core.table.try_read_attr(
 --                        local_conf, "apisix", "ssl", "fallback_sni") -- "a.test2.com"
+-- -- `nil` is returned if the attribute doesn't exist
```

Comment:
two double dashes?

Reply:
By mistake. I'll fix it

Reply:
no need to fix, The second `--` indicates that the line was originally a comment